hello.py

```diff
@@ -0,0 +1 @@
+print ("Hello World\n")
```

Comment:
need not \n, print in python print \n automatically

Reply:
Got it!I will fix it right now